### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by mitchelsellers** [dnnsoftware/Dnn.Platform#3649] [General] [answered]
Testing!

**Comment by donker** [comment]
Hey. this is cool

**Reply by WillStrohl** [reply]
Nuh uh...

**Reply by donker** [reply]
You can use the little checkmark to flag an answer

**Comment by WillStrohl** [comment]
THIS is cool.  (Not what Peter said.)

**Reply by donker** [reply]
You wish ...

**Comment by Tychodewaard** [comment]
This feature could replace the 'ideas' from the DNN corp module

**Comment by Tychodewaard** [comment]
#3647 can be referenced. Just as an example

**Comment by valadas** [comment]
This is awesome for RFCs and big requests that are not yet "developable"

**Comment by donker** [comment]
BTW, why is the nr on this 3649 if this is the first discussion?

**Reply by donker** [reply]
Ah ... under the covers it's linked to issues.

**Comment by valadas** [comment]
I think everything is an issue, like an issue, pr and discussion, so this is probably following the same numbering.

**Comment by WillStrohl** [comment]
What David said....  Everything is numbered sequentially.  The next PR/Issue will be 3650
I think the back-end is kind of like content items.

**Reply by david-poindexter** [reply]
I said something?

**Reply by valadas** [reply]
Oh yeah, don't you know we read minds ?

**Comment by thabaum** [comment, marked as answer]
So busy I missed this wow... :) look forward to discussions on issues, fun stuff!

**Comment by valadas** [comment]
I gonna mark this answered, i think it is tested now :)
